<launch>

   <node pkg="multi_robot_nav" name="dynamic_localization_client" type="dynamic_localization_client.py" ns="/tb3_0" />

</launch>